dependencies.props

```diff
@@ -30,7 +31,7 @@
     <CoreClrExpectedPrerelease>beta-25007-01</CoreClrExpectedPrerelease>
     <ExternalExpectedPrerelease>beta-25005-00</ExternalExpectedPrerelease>
     <ProjectNTfsExpectedPrerelease>beta-25007-00</ProjectNTfsExpectedPrerelease>
-    <StandardExpectedPrerelease>beta-25003-02</StandardExpectedPrerelease>
+    <StandardExpectedPrerelease>beta-25006-01</StandardExpectedPrerelease>
```

Comment:
I'd suggest updating a full package version instead (like SNI) if possible, to improve servicing behavior with major/minor/patch changes. I only see this usage, which I think could be turned into a full package version: https://github.com/dotnet/corefx/blob/edfd662f8cfb3244/external/netstandard/netstandard.depproj#L31.